js/ui/scheduler/ui.scheduler.subscribes.js

```diff
@@ -675,50 +675,44 @@ var subscribes = {
 
     convertDateByTimezone: function(date, appointmentTimezone) {
         date = new Date(date);
+        var tzOffsets = this._subscribes.getComplexOffsets(this, date, appointmentTimezone);
```

Comment:
In common case use context binding instead of passing the instance as an argument